paymentsheet/src/main/java/com/stripe/android/paymentsheet/FlowControllerCompose.kt

```diff
@@ -108,14 +110,17 @@ private fun internalRememberPaymentSheetFlowController(
         "PaymentSheet.FlowController must be created in the context of an Activity"
     }
 
-    return remember(paymentOptionCallback, paymentResultCallback) {
+    val currentPaymentOptionCallback by rememberUpdatedState(paymentOptionCallback::onPaymentOption)
```

Comment:
Would a better name for this be `sharedPaymentOptionCallback`?

Shared makes more sense to me to pass into flow controller, since it'll be updated if the merchant updates the callback.